component-catalog-connectors/connector-directory.md

```diff
@@ -5,9 +5,10 @@ The following catalog connectors should work with Elyra version 3.3 and above. C
 - To add your connector to the list [create a pull request](https://github.com/elyra-ai/examples/pulls). 
 - Learn [how to build your own catalog connector](build-a-custom-connector.md).
 
-| Connector      | Description |
-| ----------- | ----------- |
+| Connector | Description |
+| --- | --- |
 | [Apache Airflow example catalog](airflow-example-components-connector) | Provides access to a small set of curated Apache Airflow operators that you can use to get started with the Visual Pipeline Editor. |
 | [Kubeflow Pipelines example catalog](kfp-example-components-connector) | Provides access to a small set of curated Kubeflow Pipelines components that you can use to get started with the Visual Pipeline Editor. |
+| [Artifactory](artifactory-connector) | Enables Elyra to load pipelines components from a generic-type Artifactory repo. |
```

Comment:
Should mention KFP for clarity because the connector explicitly does not support Apache Airflow according to its metadata.

```suggestion
| [Artifactory](artifactory-connector) | Enables Elyra to load Kubeflow Pipelines components from a generic-type Artifactory repo. |
```

Reply:
Resolved in https://github.com/elyra-ai/examples/pull/99/commits/b97b758ec6867f83c646bb560d12cfb80f7ee7bb